components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1588,7 +1482,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
              qrcol  = qrcol*ratio
              qrheti = qrheti*ratio
              qrmul  = qrmul*ratio
-             !qrhetc = qrhetc*ratio
+             !qrhetc = qrhetc*ratio   ! AaronDonahue - Why is this commented out? We don't have qrhetc?
```

Comment:
Do we have an answer to this question yet?

Reply:
No.  I just flagged this because I thought it was strange to have it written and then commented out.  This is another tendency that isn't defined and so is 0 anyway, so commented or not it doesn't impact the solution.  Still, if in the future we do include qrhetc this seems like a potential bug.  Should we fix?  @PeterCaldwell , and thoughts?

Reply:
Yeah, this is another case of a variable that is never assigned. I think we should delete it as well, ensuring that we'll do a complete job if we decide to include it again later.

Reply:
Check, I've deleted this in the new commit.